package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@chevrotain/quick-start",
+  "description": "Your easy path to define a parser for your language based on Chevrotain DSL toolkit",
+  "version": "0.0.1",
+  "private": true,
+  "keywords": [
+    "chevrotain",
+    "starter pack",
+    "dsl",
+    "parser"
+  ],
+  "author": "Santhosh Kumar <brsanthu@gmail.com>",
+  "homepage": "https://github.com/Chevrotain/quick-start",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Chevrotain/quick-start"
+  },
+  "scripts": {
+    "clean": "rm -rf dist && rm -rf coverage && rm -rf yarn-error.log",
+    "compile": "tsc",
+    "compile:clean": "yarn clean && yarn compile",
+    "compile:watch": "tsc --watch",
+    "generate-diagram": "node scripts/generateDiagram.js",
+    "build": "rm -rf dist && tsc && yarn generate-diagram",
+    "prepublish": "yarn build",
+    "test": "ts-node -O '{\"module\":\"commonjs\"}' node_modules/jest/bin/jest.js ",
```

Comment:
jest is often sub-optimal for testing chevrotain grammars because of multiple processes combined with the chevrotain grammar's startup time (per processes)

Reply:
What is your suggestion? 

I have been using Jest in my our internal chevrotain project and it is working fine.

Reply:
I've had significant (1-2 order of magnitudes) performance improvements in running tests with Mocha. Obviously it depends n the number of tests/processes.

I may try to change this sometime this week.